job_traveler/docevents/job_traveler.py

```diff
@@ -7,13 +7,14 @@ def cutomer_part_number(item, current_cutomer):
         for cutomer in cutomers.customer_items:
             if cutomer['customer_name'] == current_cutomer:
                 return cutomer['ref_code']
+            if cutomer['ref_code'] and not cutomer['customer_name']:
```

Comment:
Please can you change `cutomer` and `cutomers` to a more appropriate name? Tainkyu